src/main/drivers/max7456.c

```diff
@@ -519,13 +517,14 @@ void max7456Brightness(uint8_t black, uint8_t white)
 //just fill with spaces with some tricks
 void max7456ClearScreen(void)
 {
-    uint16_t x;
-    uint32_t *p = (uint32_t*)&screenBuffer[0];
-    for (x = 0; x < VIDEO_BUFFER_CHARS_PAL/4; x++)
+    uint32_t * const p = (uint32_t*)&screenBuffer[0];
```

Comment:
`memset` probably contains similar optimization. And it is quite possible that gcc is calling memset anyway ..